docs/ref/contrib/admin/index.txt

```diff
@@ -1043,6 +1043,20 @@ subclass::
     If you need to specify a dynamic order (for example depending on user or
     language) you can implement a :meth:`~ModelAdmin.get_ordering` method.
 
+    .. note::
+
+        Admin changelist adds ``pk`` to ordering by default. For example if you
+        have a model like this::
+
+          from django.db import models
+
+          class Author(models.Model):
+              name = models.CharField(max_length=100)
+
+              class Meta:
+                  ordering = ['name']
+
+        The objects in admin changelist are sorted by ``name`` and ``pk``.
```

Comment:
Missing new line and we should probably mention _why_ the admin does that.

Reply:
Thank you for the comment. New line added also the reason added from
https://github.com/django/django/blob/6275b50ac26ea6c026aa7b2aae9192f5792e8a94/django/contrib/admin/views/main.py#L306